webapp/src/ts/modals/mm-modal/mm-modal.component.html

```diff
@@ -7,7 +7,7 @@ <h2>{{titleKey | translate}}</h2>
   <div class="modal-footer">
     <p class="note" *ngIf="status?.error">{{status?.error | translate}}</p>
     <a class="btn cancel" [ngClass]="{ 'disabled': status?.processing }" (click)="onCancel.emit()" *ngIf="cancelKey !== false">{{cancelKey || 'Cancel' | translate}}</a>
-    <a class="btn submit" [ngClass]="{ 'btn-primary': !danger, 'btn-danger': danger, 'disabled': disableSubmit }" (click)="onSubmit.emit()" *ngIf="submitKey && !status?.processing">{{submitKey | translate}}</a>
+    <a class="btn submit" test-id="abc" [id]="id+'-submit-btn'" [ngClass]="{ 'btn-primary': !danger, 'btn-danger': danger, 'disabled': disableSubmit }" (click)="onSubmit.emit()" *ngIf="submitKey && !status?.processing">{{submitKey | translate}}</a>
```

Comment:
Can you update the `test-id` to be the actual value you want to use and then consume it in the page object.

Reply:
Done